config.json

```diff
@@ -629,7 +629,13 @@
         "name": "Nucleotide Count",
         "uuid": "be6f3ab3-1593-4c2d-8a35-5f39c1d5c91f",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": [
+          "classes",
```

Comment:
Update: "classes" can be removed here, the exercise was changed to remove the unnecessary class definition (https://github.com/exercism/javascript/pull/1488)

Reply:
I meant to create an issue about this, good to know it has been fixed :)